### PR TITLE
Rename Person to Contact

### PR DESCRIPTION
**Pull request by bwangpj**
Rename all instances of Person to Contact in the functional and test code.

Renaming of documentation to follow soon.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any contact obtaining a copy
```
Comment (Cloud7050) [inline LICENSE:5]: We should probably not edit the license. Guessing this is overzealous replace all.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -270,7 +270,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | Priority | As a …​                         | I want to …​                                                                         | So that I can…​                                                                 |
 |----------|---------------------------------|--------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
 | `* * *`  | user                            | add contacts                                                                         | keep track of my friends and schoolmates                                        |
-| `* * *`  | user                            | delete contacts                                                                      | remove my contact with that person                                              |
+| `* * *`  | user                            | delete contacts                                                                      | remove my contact with that contact                                              |
```
Comment (Cloud7050) [inline docs/DeveloperGuide.md:273]: Wording here now sounds weird.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -280,7 +280,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | user                            | share a contact with other users or apps                                             | transfer my contact info easily                                                 |
 | `* *`    | user                            | merge duplicate contacts                                                             | my contact list stays clean                                                     |
 | `* *`    | user                            | sort contacts by certain criteria                                                    | find contacts satisfying a certain criteria easily                              |
-| `* *`    | user of many communication apps | enter info about various platforms that my contacts use                              | keep track of all the various ways I can contact the same person                |
+| `* *`    | user of many communication apps | enter info about various platforms that my contacts use                              | keep track of all the various ways I can contact the same contact                |
```
Comment (Cloud7050) [inline docs/DeveloperGuide.md:283]: Ditto.

File: src/test/java/seedu/address/storage/JsonAdaptedContactTest.java [hunk 4]
```diff
@@ -33,58 +33,58 @@ public class JsonAdaptedPersonTest {
 
     @Test
     public void toModelType_validPersonDetails_returnsPerson() throws Exception {
-        JsonAdaptedPerson person = new JsonAdaptedPerson(BENSON);
+        JsonAdaptedContact person = new JsonAdaptedContact(BENSON);
```
Comment (Cloud7050) [inline src/test/java/seedu/address/storage/JsonAdaptedContactTest.java:36]: There may be some leftover `person` variable names in the files.

**Review by Cloud7050 — CHANGES_REQUESTED** [review]
Thanks for changing all these files. I didn't read through every single file thoroughly, but I've commented on some things I noticed.

File: src/main/java/seedu/address/logic/Messages.java [hunk 5]
```diff
@@ -11,28 +11,28 @@
...
     public static final String MESSAGE_HELP_COMMAND_SHOW_HELP = "Opened help window.";
-    public static final String MESSAGE_LIST_COMMAND_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_LIST_COMMAND_SUCCESS = "Listed all contacts";
 
     // Exception message
     public static final String FILE_OPS_ERROR_FORMAT = "Could not save data due to the following error: %s";
     public static final String FILE_OPS_PERMISSION_ERROR_FORMAT =
             "Could not save data to file %s due to insufficient permissions to write to the file or the folder.";
-    public static final String MESSAGE_Duplicate_Person_Exception = "Operation would result in duplicate persons";
+    public static final String MESSAGE_Duplicate_Contact_Exception = "Operation would result in duplicate contacts";
```
Comment (rayshawntan) [inline src/main/java/seedu/address/logic/Messages.java:35]: I think we didn't catch this previously when @mamayuan edited his PR but final String variables should be in all capitalized.

**Review by rayshawntan — CHANGES_REQUESTED** [review]
Overall, I think it looks good! Just need to amend the parts Joel mentioned and the change the string name to all capital

**Review by rayshawntan — APPROVED** [review]

**Review by Cloud7050 — APPROVED** [review]

**Pull request merged**